end_to_end/end_to_end_suite_test.go

```diff
@@ -2519,6 +2519,19 @@ LANGUAGE plpgsql NO SQL;`)
 			Expect(string(output)).To(MatchRegexp(`Error loading data into table public.d: COPY d, line 1: "\d,d": ERROR: null value in column "c" violates not-null constraint`))
 			Expect(string(output)).To(MatchRegexp(`Error loading data into table public.e: COPY e, line 1: "\d,e": ERROR: null value in column "c" violates not-null constraint`))
 			Expect(string(output)).To(ContainSubstring(`Encountered 3 error(s) during table data restore`))
+			homeDir := os.Getenv("HOME")
+			helperFiles, _ := path.Glob(path.Join(homeDir, "gpAdminLogs/gpbackup_helper_*"))
+			helperCommand := exec.Command("grep", "Error copying data", helperFiles[len(helperFiles)-1])
+			helperOutput, _ := helperCommand.CombinedOutput()
+			Expect(string(helperOutput)).To(ContainSubstring(`Segment 2: Oid 16397, Batch 0: Error encountered: Error copying data: write`))
```

Comment:
Can you remove oid from these lines?

Reply:
oid is taken from backup and always same